docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance.cs

```diff
@@ -5,57 +5,22 @@
 
 namespace Microsoft.ML.Samples.Dynamic
 {
```

Comment:
IMO we should leave the samples in this project small and simple; they are meant to illustrate the API usage; not give a complete overview/tutorial on how to solve  a problem. 
Longer, more complete samples can go in the machinelarning-samples repo. 

It might be overwhelming, and hard to digest, to see a long, long example in a page that is supposed to just tell you how to call the API.

Reply:
I agree with this. However, `PFI` is a bit challenging because the return values also need to be explained. What I've done here is try to make these short and easy, while showing how to use the API and interpret the results.

The latest commits (with rebase) refactor each task (e.g. `Regression`, `BinaryClassification`) into its own short file, adding a `PfiHelper` class to hold common things like data loading and linear-model-weight extraction.

One thing that adds to the length is a discussion of how to interpret the results. I believe these are appropriate for the API because these are the first sorts of questions people ask when you present PFI to them.

What do you think @sfilipi  @montebhoover  and @glebuk ?

Reply:
I like this - the separation of the files makes each sample more manageable.  I'd be curious if the prose explanation at the end of the sample would be better suited to show up in the docs outside of the example code, like in some sort of remarks section, but that is a minor improvement in my mind.